js/actions/DeploymentActions.js

```diff
@@ -0,0 +1,59 @@
+var oboe = require("oboe");
```

Comment:
I like the clarity and simplicity of Obeo and it's smal size of 5K.

The usage is pretty straight forward and well readable:
```
oboe({
   url: String,
   method: String,          // optional
   headers: Object,         // optional
   body: String|Object,     // optional
   cached: Boolean,         // optional
   withCredentials: Boolean // optional, browser only
})
```
http://oboejs.com/api

Also I like the Promises approach with (.done, .fail, etc.) and it has filtering off data on the fly (streams!) on top what we could need in the future.

We defined a Key Result:
```
Should be usable at big scale
```
so this could be interessing
```
Step outside the trade-off between big and small JSON

Oboe.js breaks out of the tradeoff by beating both. Large resources load just as responsively as smaller ones so the developer can request more and let it stream. 
```